containers/compilers.yaml

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    name: "intel",
+    package: "intel-oneapi-compilers",
+    version: "2021.1.2",
+  },
+]
```

Comment:
I definitely want the next PR to add a compiler to this and check it works ;)

Reply:
Will give that a go!